Unix/tests/cli/test_cli.cpp

```diff
@@ -151,6 +151,32 @@ static int StartServer()
     travisCI = true;
 #endif
 
+    //
+    // kill running omiserver if have any
+    //
+
+    
+    {
+        char buf[64];
+
+        int pid = -1;
+        int status = PIDFile_Read(&pid);
+
+        if (pid != -1)
+        {
+
+#if defined(aix)
+            sprintf(buf, "sudo kill -TERM %d", pid);
+#elif !defined(solaris) && !defined(linux)
+            sprintf(buf, "killall -TERM %d", pid);
+#else
+            sprintf(buf, "pkill -TERM %d", pid);
+#endif
+            system(buf);
+            Sleep_Milliseconds(10);
```

Comment:
I'm pretty uncomfortable with a sleep like this. First, 50 milliseconds is very quick, and I can virtually guarantee that, at times, some of our slower platforms may not keep up.

For this sort of thing, I like do do something like (pseudo-code):

```
int counter = 0;
<tell process to shut down>
while (counter <= 50 && <process still exists>) {
    Sleep_Milliseconds(100);
    counter++;
}

if <process still exists> {
    log error and perhaps abort or try again with kill -9
}
```

In this way, if the process actually goes away quickly, you end quickly. If OMI takes some time to clean up, that's okay too - with 100 milliseconds sleep and a counter limit of 50, you'll have 5 full seconds to shut down, which should normally work.

We've had a LOT of issues over all our platforms with intermittent failures for tests that sleep. Much better to verify that things worked or wait longer.

Also note that, sometimes (I don't know why), we've observed the inability to kill processes, even with -9, particularly on AIX. We ended up rebooting when that occurred, and after that it wouldn't repro.